.github/scripts/run-config.py

```diff
@@ -130,6 +130,52 @@ def run_bmconfig(config: BenchmarkModelConfig, repo_path: Path, output_path: Pat
         return
     subprocess.check_call(cmd, cwd=repo_path)
 
+def run_bmconfig_profiling(config: BenchmarkModelConfig, repo_path: Path, output_path: Path, dryrun=False):
+    nsys_path = "/opt/nvidia/nsight-systems/2022.2.1/bin/nsys"
```

Comment:
I would prefer to add nsys binary path to PATH and emit a warning if not exists. It must be good to provide a message how to install nsys as well.

Reply:
Done.